backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/llvm/LlvmDeclarations.kt

```diff
@@ -41,8 +41,7 @@ internal class LlvmDeclarations(
     private val fields: Map<IrField, FieldLlvmDeclarations>,
     private val staticFields: Map<IrField, StaticFieldLlvmDeclarations>,
     private val unique: Map<UniqueKind, UniqueLlvmDeclarations>) {
-    fun forFunction(function: IrFunction) = functions[function] ?:
-            error("${function.descriptor} in ${function.parent.fqNameSafe}")
+    fun forFunction(function: IrFunction) = functions[function]
```

Comment:
So here signature is silently changed. Maybe add another one? Or smth like
```
fun forFunction(function: IrFunction, orElse: (IrFunction) -> Unit = {
  it -> error("${it.descriptor} in ${it.parent.fqNameSafe}")}) = 
  functions[function] ?: orElse(function)
```

Reply:
Then signature will change even more dramatically. But I understand idea and will try to develop to something useful